Assignment2/NicolásOrtizSuaña.ttl

```diff
@@ -0,0 +1,18 @@
+@base <http://www.oeg-upm.net/ontologies/class/ >
+@prefix class: <http://www.ontologies.org/ontologies/class#>
+@prefix computer: <http://www.ontologies.org/ontologies/computer#>
+@prefix user: <http://www.ontologies.org/ontologies/user#>
+@prefix sensor: <http://www.ontologies.org/ontologies/sensor#>
+@prefix measurement: <http://www.ontologies.org/ontologies/measurement#>
+
+:Class01          class:includes :Sensor029;
+                  class:includes :Computer101.
+
+:Measurement8401  measurement:hasTemperature :"29";
```

Comment:
The symbol of a colon before the number 29 does not make sense. Please validate this RDF.